lib/sda/node.go

```diff
@@ -227,6 +286,16 @@ func (n *Node) RegisterHandler(c interface{}) error {
 	return nil
 }
 
+// RegisterHandlers registers a list of given handlers by calling RegisterHandler above
+func (n *Node) RegisterHandlers(handlers []interface{}) error {
```

Comment:
Stylistic remark: Instead of slices, why not using `varargs` ? It avoids using a slice of handlers in the protocols.